### PR TITLE
Fix/popover render

### PR DESCRIPTION
**Pull request by VictorOhashi**
Made some changes to fix issue #53 based on [Flutter tutorial of synchronously BuildContexts](https://www.youtube.com/watch?v=bzWaMpD1LHY&t=289), and some adjustments to null safety and animations.

## Pre-launch Checklist

- [x] I read and followed the [Effective Dart](https://dart.dev/guides/language/effective-dart/style).
- [x] I listed at least one issue or feature that this PR fixes or adds in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I updated/added relevant documentation to README.md.
- [ ] I updated/added relevant code samples to [example](https://github.com/minikin/popover/blob/main/example/lib/main.dart).
- [ ] I added new tests to check the change I am making or feature I am adding, or Minikin said the PR is test-exempt.
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by minikin** [comment]
@VictorOhashi Thank you for your contribution! I'll review PR this weekend.

**Comment by VictorOhashi** [comment]
@minikin any updates?

File: lib/src/popover.dart [hunk 1]
```diff
@@ -96,7 +92,6 @@ Future<T?> showPopover<T extends Object?>({
   double? width,
   double? height,
   VoidCallback? onPop,
-  bool Function()? isParentAlive,
```
Comment (minikin) [inline lib/src/popover.dart:99]: We should consider using `@Deprecated` here instead of introducing breaking changes.

**Review by minikin — COMMENTED** [review]
Overall looks good. 
Would you mind adding a video of a screen recording for the example app with these changes?

**Comment by VictorOhashi** [comment]
> Overall looks good. Would you mind adding a video of a screen recording for the example app with these changes?

Everything working well, included portrait change.

https://user-images.githubusercontent.com/38299943/207369532-82cb476e-bc7c-423c-b3d8-464504404c3c.mp4

**Comment by angelomedeiros** [comment]
Amazing job!

**Pull request merged**